pkg/cmd/browse/browse_test.go

```diff
@@ -228,6 +228,35 @@ func Test_runBrowse(t *testing.T) {
 			baseRepo:    ghrepo.New("bchadwic", "LedZeppelinIV"),
 			expectedURL: "https://github.com/bchadwic/LedZeppelinIV/tree/trunk/main.go",
 		},
+		{
+			name: "branch flag within dir",
+			opts: BrowseOptions{
+				Branch:           "feature-123",
+				PathFromRepoRoot: func() string { return "pkg/dir" },
+			},
+			baseRepo:    ghrepo.New("bstnc", "yeepers"),
+			expectedURL: "https://github.com/bstnc/yeepers/tree/feature-123/",
```

Comment:
Minor: let's generate this URL without a trailing slash, since that's GitHub web would also do it for branches but no path.

Reply:
This is a great point and is the current behavior of using the branch flag even at the repo's root dir. As a new contributor on this repo, do we prefer to fix it here or create a new issue to track it properly?

Reply:
Oh, if the old code also generated the trailing slash, then you're right that it should be changed in a PR separate from this one.  And even if it stays, it's not a big deal. 👍